aten/src/ATen/native/cudnn/MHA.cpp

```diff
@@ -487,7 +487,11 @@ std::unique_ptr<fe::graph::Graph> build_graph(
   auto scaled_dot_product_flash_attention_options =
       fe::graph::SDPA_attributes()
           .set_name("CUDNN_SDPA")
+#if defined(CUDNN_FRONTEND_VERSION) && CUDNN_FRONTEND_VERSION == 11200
+          .set_is_inference(!return_softmaxstats)
```

Comment:
This should be `CUDNN_FRONTEND_VERSION <= 11200`, and no need to use `defined(CUDNN_FRONTEND_VERSION)` as this entire block should be enclosed by that